src/display/editor/freetext.js

```diff
@@ -861,6 +861,9 @@ class FreeTextEditor extends AnnotationEditor {
   renderAnnotationElement(annotation) {
     const content = super.renderAnnotationElement(annotation);
     if (this.deleted) {
+      annotation.updateEdited({
+        popupDeleted: true,
```

Comment:
`popupDeleted` no longer exists after the recent change, so this should probably be `deleted` now instead.